src/main.js

```diff
@@ -16,7 +16,7 @@ const { onConnect, onFirefoxConnect } = require("./utils/client");
 const reducers = require("./reducers");
 const selectors = require("./selectors");
 
-const App = require("./components/App");
+const App = require("./components/App").default;
```

Comment:
If you use import/export for the `App` file I think changing this line  to
`import App from './components/App'` isn't bad.